receiver/hostmetricsreceiver/testdata/config.yaml

```diff
@@ -11,9 +11,9 @@ receivers:
       filesystem:
       memory:
       network:
-        include:
-          interfaces: ["test1"]
-          match_type: "strict"
+      #        include:
+      #          interfaces: ["test1"]
+      #          match_type: "strict"
```

Comment:
Is this intentional change or debug?

Reply:
It was for debugging. However, if I add this back; the test fails as i can't import `networkscraper/internal/metadata` here

Reply:
I think something  like this should do the trick
```
networkscraper.TypeStr: func() *networkscraper.Config (
     cfg := (&networkscraper.Factory{}).CreateDefaultConfig(),
     cfg.Include = networkscraper.MatchConfig{
         Interfaces: []string{"test1"},
         Config: filterset.Config{MatchType: "strict"},
     },
}(),
```

Reply:
Tried that keep seeing this

Reply:
Oh, I set a wrong type of the return value. Updated the snipped. Should work now

Reply:
I tried both `internal.Config` and `networkscraper.Config` both don't work

```cfg.Include undefined (type "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal".Config has no field or method Include)compilerMissingFieldOrMethod``` - if return type is `internal.Config`. 

Tried forcing some type conversions/casting as well; didn't work. 
I am missing something super silly here.

Reply:
Alright this worked